nucypher-core-python/src/lib.rs

```diff
@@ -720,6 +733,71 @@ impl ThresholdDecryptionRequest {
     }
 }
 
+//
+// E2EEThresholdDecryptionRequest
+//
+#[pyclass(module = "nucypher_core")]
+#[derive(derive_more::From, derive_more::AsRef)]
+pub struct E2EEThresholdDecryptionRequest {
+    backend: nucypher_core::E2EEThresholdDecryptionRequest,
+}
+
+#[pymethods]
+impl E2EEThresholdDecryptionRequest {
+    #[getter]
+    pub fn decryption_request(&self) -> ThresholdDecryptionRequest {
+        self.backend.decryption_request.clone().into()
```

Comment:
We should be able to skip `clone()` here

```suggestion
        self.backend.decryption_request.into()
```